invenio_rdm_records/services/errors.py

```diff
@@ -154,9 +153,9 @@ def __init__(self, record_id, community_id):
     def description(self):
         """Exception description."""
         return _(
-            "The record {record_id} in not included in the community {community_id}.".format(
-                record_id=self.record_id, community_id=self.community_id
-            )
+            "The record %(rec_id)s in not included in the community %(com_id)s.",
+            rec_id=self.record_id,
+            com_id=self.community_id,
```

Comment:
The original discussion regarding translations can be found [here](https://github.com/inveniosoftware/invenio-rdm-records/pull/1450#discussion_r1318892078)